mne/evoked.py

```diff
@@ -841,9 +841,10 @@ def get_peak(self, ch_type=None, tmin=None, tmax=None, mode='abs',
             The time point of the maximum response, either latency in seconds
             or index.
         """
-        supported = ('mag', 'grad', 'eeg', 'misc', 'None')
+        supported = ('mag', 'grad', 'eeg', 'seeg', 'misc', 'None')
 
-        data_picks = pick_types(self.info, meg=True, eeg=True, ref_meg=False)
+        data_picks = pick_types(self.info, meg=True, eeg=True, seeg=True,
+                                ref_meg=False)
         types_used = set([channel_type(self.info, idx) for idx in data_picks])
```

Comment:
You still need to add the ``seeg`` channel to lines 865-878. And while we are at it, perhaps the docstrings could be updated for the ``None`` parameters.